src/i2p.cpp

```diff
@@ -150,7 +150,7 @@ bool Session::Accept(Connection& conn)
                 throw std::runtime_error("wait on socket failed");
             }
 
-            if ((occurred & Sock::RECV) == 0) {
+            if (occurred == 0) {
                 // Timeout, no incoming connections within MAX_WAIT_FOR_IO.
```

Comment:
Maybe update this to "no incoming connections or errors" to match the code change.

Reply:
Done